packages/framework/azure-client/package.json

```diff
@@ -64,7 +64,7 @@
     "mocha": "^8.4.0",
     "rimraf": "^2.6.2",
     "start-server-and-test": "^1.11.7",
-    "tinylicious": "^0.4.34614",
+    "tinylicious": "^0.4.38350",
```

Comment:
```suggestion
    "@fluidframework/azure-local-service": "^0.1.38773",
```

Reply:
^^ This would also require a change to the `start:tinylicious:test` script. Can be a followup.

Reply:
updated here.  i kept the script names for now, since other packages are still using the tinylicious naming